test/bats/azure-test.bats

```diff
@@ -142,6 +147,8 @@ SLEEP_TIME=1
     assert_success
     sleep 5
 
+    run kubectl apply -f ./config/samples/config_v1alpha1_verifier_cosign.yaml
```

Comment:
thanks for catching this Binbin. Are the azure test and c8 test mostly the same step? I m curious is there any way to share the script between these two tests.

Reply:
yeah that's a good point. Reason I separated it into it's own bats file was that the ACR images we use are different and they don't correspond exactly to the images used in the local registry one. We could share it but then we'd be setting alot of extra environment variables for AKS and non AKS workflows.

Reply:
THanks for the insight Akash. Not a blocker for this PR, just something to think about in the next iteration of test improvements

Reply:
right, my next plan is to build those images and push to ACR in each test instead of pulling the wabbitnetworks one. In this way we could share a lot with local tests.